src/form/sections/HeaderFooterMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from '@folio/stripes/components';
+import React from 'react';
+import { FieldArray } from 'react-final-form-arrays';
+import { HeaderFooterTokenType } from '../../types/TokenTypes';
+import HeaderFooterCard from '../../components/Token/HeaderFooter/HeaderFooterCard';
+import { FormattedMessage } from 'react-intl';
+
+export default function HeaderFooterMenu({ name }: { name: string }) {
```

Comment:
it's not clear what this component does, according to a name it's a menu with some relation to a header/footer, but in fact it's field array, how is it connected to menu?

Reply:
`HeaderFooterSection` still says nothing

Reply:
I mean it's the part of the form that configures the header/footer data, I am not sure what else can be a better name.